.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: Build & Tests
+
+on:
+  push:
+  pull_request:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build_test:
+    # Used for installing miri
+    # https://rust-lang.github.io/rustup-components-history/
+    env:
+      TOOLCHAIN_VERSION: '2022-09-26'
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        channel: [ "stable", "beta", "nightly" ]
+        target: [ "x86_64-unknown-linux-gnu" ]
+        features: [ "" , "--features alloc,simd,simd-nightly"]
+
+    # Prevents a workflow run from failing when a job fails
+    continue-on-error: ${{ matrix.channel != 'stable' }}
```

Comment:
I think this is actually too permissive. AFAICT, the only times we want to continue on error are:
- We're compiling for a non-x86 or x86_64 target and running `cargo test`
- We're compiling for non-nightly and using the `simd-nightly` feature

Any other failure - even if on non-stable - should "count".

Reply:
Ah, I see your conditional logic below for running `cargo test` on the wrong target, so I guess it's just the `simd-nightly` issue.

Reply:
Should I make a case where I only use `simd-nightly` for this or is using all features fine?